backend/fms_core/models/full_sample.py

```diff
@@ -0,0 +1,80 @@
+from django.db import models
+from .container import Container
+from .derived_sample import DerivedSample
+from .biosample import Biosample
+from .sample_kind import SampleKind
+from .project import Project
+from .sample import Sample
+from .process_measurement import ProcessMeasurement
+from django.contrib.auth.models import User
+from django.contrib.postgres.fields import ArrayField
+
+__all__ = ["FullSample"]
+
+
+class FullSample(models.Model):
+    """ Class to provide information about a sample as a view. """
+    sample_kind = models.ForeignKey(SampleKind, on_delete=models.DO_NOTHING, db_column='sample_kind_id',
```

Comment:
I dont think those on_delete attributes are necessary on FullSample attributes. This is a read-only model, so theoretically there should be not deletion operation performed

https://stackoverflow.com/a/35780859

Reply:
![Screen Shot 2021-11-10 at 2 31 26 PM](https://user-images.githubusercontent.com/33667377/141181672-44ec56aa-3aaf-4d3a-aef6-ca7cc60d2a82.png)

Django does complain if the attributes are not there